frontend/src/views/GameView.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <div class="game-view container">
-    <div class="row py-0" style="height: 100%; max-height: 100%">
+    <div
+      v-if="!hideChat"
```

Comment:
if you do `v-if="!hideChat"` here, you can do `v-else` at line 19 instead of `v-if="hideChat`